tests/integration/targets/nxos_lag_interfaces/defaults/main.yaml

```diff
@@ -1,2 +1,2 @@
 ---
-testcase: '*'
+testcase: '[^_].*'
```

Comment:
need to check for empty config in rendered as well.

Reply:
That test is already there in empty_config.yaml. Do you mean something else?